docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,774 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ibis for dplyr Users"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[R](https://www.r-project.org/) users familiar with [dplyr](https://dplyr.tidyverse.org/) and other packages in the [Tidyverse](https://www.tidyverse.org/) are likely find Ibis familiar.\n",
+    "In fact, some Ibis features were even inspired by similar features in the [Tidyverse](https://www.tidyverse.org/).\n",
+    "\n",
+    "However, due to differences between Python and R and the design and goals of Ibis itself, you may notice some big differences right away:\n",
+    "\n",
+    "- **No pipe:** The handy [magrittr pipe](https://magrittr.tidyverse.org/) (`%>%`) or R's newer native pipe (`|>`) don't exist in Python so you instead have to chain sequences of operations together with a period (`.`).\n",
```

Comment:
Is it maybe worth explaining here that `.` with Python objects is like `$` with S3 objects R? (It is the operator used to access attributes and methods of an object.)

Reply:
Sure, I think it's safe to assume a person who has only used R/tidyverse will find `.` is new syntax.